doc/md/schema-fields.md

```diff
@@ -613,6 +613,168 @@ func (User) Fields() []ent.Field {
 }
 ```
 
+## Enum Fields
+
+Use Ent API to generate Enum type and define possible values:
+```go
+// Fields of the User.
+func (User) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("first_name"),
+		field.String("last_name"),
+		field.Enum("size").
+			Values(
+				"big",
+				"small",
+			),
+	}
+}
+```
+
+When custom type that is convertible to the Go basic type:
+
+```go
```

Comment:
Merge the 2 code blocks to one. Easier to follow.

Reply:
I actually like the split, looks more like a recipe and easier for the reader to understand what is "in ent" and what Is part of my own type (outside ent).